lib/puppet/type/cisco_ospf_area.rb

```diff
@@ -124,6 +136,44 @@ def name
     munge { |value| value == 'default' ? :default : value }
   end # property filter_list_out
 
+  newproperty(:nssa) do
+    desc 'Defines the area as NSSA (not so stubby area). This is
+          mutually exclusive with stub and stub_no_summary.'
```

Comment:
done with fixes for all comments. Also created another PR for NU for the nssa_set method changes : https://github.com/cisco/cisco-network-node-utils/pull/434